src/com/theironyard/novauc/Wealth.java

```diff
@@ -0,0 +1,17 @@
+package com.theironyard.novauc;
+
+/**
+ * Created by octavio on 2/7/17.
+ */
+public class Wealth {
+        String Occupation;
+        String Company;
+        int yearsEmployed;
+        int salary;
+        boolean loveJob;
+
```

Comment:
Octavio great commit message. Great habit to get into.  Variables follow the format of lowercase letter to start the first word and uppercase to start the second word.  Use line 11 as an example.  Overall great job I like the work you did in the main class.